index.html

```diff
@@ -245,35 +269,57 @@ <h1>About Me</h1>
                             </div>
                             
                     </div>
-                </section>
+                </div>
+            </section>
                 <section class="details">
                 
                     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
                 
                 <form class="form-style" action="https://formspree.io/f/mnqwzknj" method="POST">
-                   
-                    <input type="text" maxlength="30" placeholder="Full Name" required>
-            
-                
-                    <input type="email" name="email" placeholder="Enter address" required>
-            
+                   <div class="form-style">
+                       <div class="input">
+                    <input type="text" maxlength="30" placeholder="First Name" required>
+                    <input type="text" maxlength="30" placeholder="Last Name" required>
```

Comment:
- [ ] In this place, I noticed that. Your `Last Name` input is slightly smaller than the original design. As you can see in this image. Please make it equal width with other inputs. Such as, `Enter address`, `Enter text here`...

![Screen Shot 2022-04-02 at 4 31 05 PM](https://user-images.githubusercontent.com/69011963/161381260-1a9fdf47-af9c-4772-8982-90485fd87751.png)

Reply:
please kindly refresh your Browser, its the same  length from my end thanks

Reply:
done,

Reply:
Send me a screenshot I wanna see?